definitions/npm/react-router-dom_v6.x.x/flow_v0.104.x-/test_react-router-dom.js

```diff
@@ -484,5 +485,54 @@ describe("react-router-dom", () => {
         sensitive: 'foo',
       });
     });
+
+    it('useMatches', () => {
+      type Params<Key: string> = {
+        +[key: Key]: string | void,
+      };
+
+      type Matches = Array<{|
+        id: string,
+        pathname: string,
+        params: Params<string>,
+        data: mixed,
+        handle: {|
+          custom: string,
+        |},
+      |}>;
+      const matches: Matches = useMatches();
+
+      type MatchesWithHandle = Array<{|
```

Comment:
But it's the same as `Matches`

Reply:
hmm yes but with refinement. Well, I'm not quiet sure what is relevant to test.

Reply:
"refinement"? 

Well no big deal here anyway.